server/src/test/java/com/example/tyfserver/payment/controller/PaymentControllerTest.java

```diff
@@ -0,0 +1,112 @@
+package com.example.tyfserver.payment.controller;
+
+import com.example.tyfserver.auth.config.AuthenticationArgumentResolver;
+import com.example.tyfserver.auth.config.AuthenticationInterceptor;
+import com.example.tyfserver.member.exception.MemberNotFoundException;
+import com.example.tyfserver.payment.domain.Payment;
+import com.example.tyfserver.payment.dto.PaymentSaveRequest;
+import com.example.tyfserver.payment.dto.PaymentSaveResponse;
+import com.example.tyfserver.payment.exception.PaymentSaveRequestException;
+import com.example.tyfserver.payment.service.PaymentService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.restdocs.AutoConfigureRestDocs;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.mockito.Mockito.*;
+import static org.springframework.restdocs.mockmvc.MockMvcRestDocumentation.document;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.post;
+import static org.springframework.restdocs.operation.preprocess.Preprocessors.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(controllers = PaymentController.class)
+@AutoConfigureRestDocs
+public class PaymentControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @MockBean
+    PaymentService paymentService;
```

Comment:
```suggestion
    private PaymentService paymentService;
```
접근제한자를 default로 둔 이유가 따로 있는게 아니라면 private로 변경하는게 좋을 것 같아
(아래의 필드도 마찬가지로!!)